pages/en/lb3/3.0-Release-Notes.md

```diff
@@ -2,49 +2,76 @@
 title: "3.0 Release Notes"
 lang: en
 layout: page
+toc: false
 keywords: LoopBack
 tags:
 sidebar: lb3_sidebar
 permalink: /doc/en/lb3/3.0-Release-Notes.html
 summary:
 ---
 
-{% include note.html content="
-LoopBack 3.0 has not yet been released. These release notes reflect the current state of the project, and may change before release.
+{% include note.html content="LoopBack 3.0 is currently in pre-release. These release notes reflect the current state of the project, and are still changing.  For more information, see [Announcing LoopBack 3.0 Release Candidate](https://strongloop.com/strongblog/announcing-loopback-3-0-release-candidate/).
 " %}
 
-Refer to [3.0 Migration Guide](http://loopback.io/doc/en/lb3/Migrating-to-3.0.html)
-for instructions on upgrading existing LoopBack 2.x applications to the new
-version.
+{% include toc.html %}
 
-## loopback-datasource-juggler was moved from peerDependencies to dependencies
+For instructions on upgrading a LoopBack 2.x application, see the [3.0 Migration Guide](Migrating-to-3.0.html).
 
-Originally, we (ab)used peer dependencies to ensure there is only one instance
-of loopback-datasource-juggler in the dependency tree, so that there is only
-one singleton instance of model registry. This was very fragile and might not
+## Changes to loopback-datasource-juggler
+
+### loopback-datasource-juggler is now a regular dependency
```

Comment:
This change was made in `loopback`, not in `loopback-datasource-juggler`. I am not sure whether it makes sense to list it under "Changes to loopback-datasource-juggler"?


Also I would add "of loopback" at the end of the title, to prevent confusion.

```
loopback-datasource-juggler is now a regular dependency of loopback
```